packages/expo-localization/android/src/main/java/expo/modules/localization/LocalizationModule.kt

```diff
@@ -70,14 +70,28 @@ class LocalizationModule : Module() {
     // These keys are used by React Native here: https://github.com/facebook/react-native/blob/main/React/Modules/RCTI18nUtil.m
     // We set them before React loads to ensure it gets rendered correctly the first time the app is opened.
     val supportsRTL = appContext.reactContext?.getString(R.string.ExpoLocalization_supportsRTL)
-    if (supportsRTL != "true" && supportsRTL != "false") return
-    context
-      .getSharedPreferences(SHARED_PREFS_NAME, Context.MODE_PRIVATE)
-      .edit()
-      .also {
-        it.putBoolean(KEY_FOR_PREFS_ALLOWRTL, supportsRTL == "true")
-        it.apply()
+    val forcesRTL = appContext.reactContext?.getString(R.string.ExpoLocalization_forcesRTL)
+
+    if (forcesRTL == "true") {
+      context
+        .getSharedPreferences(SHARED_PREFS_NAME, Context.MODE_PRIVATE)
+        .edit()
+        .also {
+          it.putBoolean(KEY_FOR_PREFS_ALLOWRTL, true)
+          it.putBoolean(KEY_FOR_PREFS_FORCERTL, true)
+          it.apply()
+        }
+    } else {
+      if (supportsRTL == "true" || supportsRTL == "false") {
+        context
+          .getSharedPreferences(SHARED_PREFS_NAME, Context.MODE_PRIVATE)
+          .edit()
+          .also {
+            it.putBoolean(KEY_FOR_PREFS_ALLOWRTL, supportsRTL == "true")
```

Comment:
Should we also set `KEY_FOR_PREFS_FORCERTL` to `false`? This key may already exist in the shared preferences. We don't want to use the old value.

Reply:
I'll do that but only if it's explicitly `false`

```
            if (forcesRTL == "false") {
              it.putBoolean(KEY_FOR_PREFS_FORCERTL, false)
            }
```